examples/theano/cluster_demo.py

```diff
@@ -1,57 +0,0 @@
-import sys
```

Comment:
it would be nice to port this example to TF since I don't think there's an example of complex run_experiment usage there.